### DISCUSSION TITLE
Anyone else have trouble with debugging breakpoints in API routes?

### DISCUSSION THREAD

**Discussion by tettoffensive** [vercel/next.js#17119] [Help]
I can only occasionally get vscode to break where I've set my breakpoints in my API routes. Sometimes, I click to set breakpoints and they don't set at all. Other times, they set but on a different line. And other times, the code breaks where I haven't set a breakpoint at all. If I edit my code at all, then I can forget about breakpoints working at all. My only hope to fix it is to restart my server, quit everything and relaunch.

I imagine this has something to do with webpack (I've had issues debugging vue which also uses webpack). But I have to believe there is something wrong with my setup because if everyone was having issues debugging like I am, I'd be finding complaints everywhere.

**I'm using vscode and have my launch.json set to:**
```json
{
  "type": "node",
  "request": "attach",
  "name": "Attach to Program",
  "skipFiles": ["<node_internals>/**"],
  "timeout": 60000,
  "port": 9229
}
```

I'm running `yarn dev` where: `"dev": "NODE_OPTIONS='--inspect' next dev",`

Next.js v9.5.1

Am I missing something? Do others have this issue?

**Comment by fcFn** [comment]
Absolutely have this issue in Next v11.1.2. Any advice?
